src/EnergyPlus/Plant/PlantManager.cc

```diff
@@ -3263,6 +3263,9 @@ void SizePlantLoop(EnergyPlusData &state,
             Real64 DesignPlantCapacity =
                 cp * FluidDensity * state.dataSize->PlantSizData(PlantSizNum).DesVolFlowRate * state.dataSize->PlantSizData(PlantSizNum).DeltaT;
             state.dataSize->PlantSizData(PlantSizNum).DesCapacity = DesignPlantCapacity; // store it for later use in scaling
+            if (state.dataPlnt->PlantFinalSizesOkayToReport) {
+                BaseSizer::reportSizerOutput(state, "PlantLoop", state.dataPlnt->PlantLoop(LoopNum).Name, "Design Capacity [W]", DesignPlantCapacity);
+            }
```

Comment:
New EIO output which is useful to check the chilled water plant loop PLR used in the `ModulatedLoopPLR` control option.